cartridges/cloudinary_sg_changes/cartridge/controllers/Product.js

```diff
@@ -51,6 +51,7 @@ function show() {
             cloudinary.videoEnabled = isVideoEnabled;
             cloudinary.videoPlayerEnabled = isVideoPlayerEnabled;
             cloudinary.cloudName = cloudinaryConstants.CLD_CLOUDNAME;
+            cloudinary.domain = cloudinaryConstants.CLD_BASE_PATH.match(/^(?:https?:\/\/)?([^\/]+)\/?/)[1];
```

Comment:
This regex will fail with custom ports and when trailing slash will not be provided.
Please use this regex: `^(?:https?:\/\/)?([^\/:]+)`

Reply:
the `\/?` at the end of the regex doesn't add more information. please use
`/^(?:https?:\/\/)?([^\/]+)/`